content/PGP/Importing_keys.adoc

```diff
@@ -289,5 +289,5 @@ And as a last step we've now moved the Authentication key to the YubiKey.
   gpg> quit
   Save changes? (y/N) y
 
-After this the keyring is saved, now no longer containing the real secret key,
+After this the keyring is saved, and no longer contains the real secret key,
```

Comment:
I still think this sentence is a bit awkward. How about: "After this the keyring is saved. At that point it no longer contains the real secret key, only a pointer indicating that it's stored on a smart card." (note smart card rather than smartcard).